platform/solr/solr-xpath/src/main/java/org/codice/solr/xpath/XpathQueryParser.java

```diff
@@ -52,15 +51,6 @@ protected Query getFieldQuery(String field, String queryText, int slop) throws S
    * @return Lucene query to pre-filter using xpath index
    */
   private Query getLuceneQuery(final String queryText) {
-    String xpath = queryText;
-
-    // Assume root is context node since evaluation does not have a context item
-    if (StringUtils.startsWith(xpath, "./")) {
-      xpath = StringUtils.removeStart(xpath, ".");
-    } else if (!StringUtils.startsWith(xpath, "/")) {
-      xpath = "/" + xpath;
-    }
-
     return null; // TODO DDF-1882 add lucene xpath pre-filtering
```

Comment:
❓ Should we not just commented out this code for use when the TODO is finally Worked on?